src/api/rpc/full_node/index.ts

```diff
@@ -503,6 +503,162 @@ export async function get_fee_estimate<T extends TRPCAgent | TDaemon>(agent: T,
   return agent.sendMessage<R>(chia_full_node_service, get_fee_estimate_command, data);
 }
 
+
+
+// simulator RPCs
+export const get_all_blocks_command = "get_all_blocks";
+export type get_all_blocks_command = typeof get_all_blocks_command;
+export type TGetAllBlocksRequest = {
+};
+export type TGetAllBlocksResponse = {
+  blocks: FullBlock[];
+  success: bool;
+};
+export type WsGetAllBlocksMessage = GetMessageType<chia_full_node_service, get_all_blocks_command, TGetAllBlocksResponse>;
+export async function get_all_blocks<T extends TRPCAgent | TDaemon>(agent: T, data: TGetAllBlocksRequest){
+  type R = ResType<T, TGetAllBlocksResponse, WsGetAllBlocksMessage>;
+  return agent.sendMessage<R>(chia_full_node_service, get_all_blocks_command, data);
+}
+
+
+
+export const farm_block_command = "farm_block";
+export type farm_block_command = typeof farm_block_command;
+export type TFarmBlockRequest = {
+  address: str;
+  guarantee_tx_block?: bool;
+  blocks?: bool;
+};
+export type TFarmBlockResponse = {
+  new_peak_height: uint64;
+  success: bool;
+};
+export type WsFarmBlockMessage = GetMessageType<chia_full_node_service, farm_block_command, TFarmBlockResponse>;
+export async function farm_block<T extends TRPCAgent | TDaemon>(agent: T, data: TFarmBlockRequest){
+  type R = ResType<T, TFarmBlockResponse, WsFarmBlockMessage>;
+  return agent.sendMessage<R>(chia_full_node_service, farm_block_command, data);
+}
+
+
+
+export const set_auto_farming_command = "set_auto_farming";
+export type set_auto_farming_command = typeof set_auto_farming_command;
+export type TSetAutoFarmingRequest = {
+  auto_farm: bool;
+};
+export type TSetAutoFarmingResponse = {
+  auto_farm_enabled: bool;
+  success: bool;
+};
+export type WsSetAutoFarmingMessage = GetMessageType<chia_full_node_service, set_auto_farming_command, TSetAutoFarmingResponse>;
+export async function set_auto_farming<T extends TRPCAgent | TDaemon>(agent: T, data: TSetAutoFarmingRequest){
+  type R = ResType<T, TSetAutoFarmingResponse, WsSetAutoFarmingMessage>;
+  return agent.sendMessage<R>(chia_full_node_service, set_auto_farming_command, data);
+}
+
+
+
+export const get_auto_farming_command = "get_auto_farming";
+export type get_auto_farming_command = typeof get_auto_farming_command;
+export type TGetAutoFarmingRequest = {
+};
+export type TGetAutoFarmingResponse = {
+  new_peak_height: uint64;
+  success: bool;
+};
+export type WsGetAutoFarmingMessage = GetMessageType<chia_full_node_service, get_auto_farming_command, TGetAutoFarmingResponse>;
+export async function get_auto_farming<T extends TRPCAgent | TDaemon>(agent: T, data: TGetAutoFarmingRequest){
+  type R = ResType<T, TGetAutoFarmingResponse, WsGetAutoFarmingMessage>;
+  return agent.sendMessage<R>(chia_full_node_service, get_auto_farming_command, data);
+}
```

Comment:
Should be
```typescript
export async function get_auto_farming<T extends TRPCAgent | TDaemon>(agent: T){
  type R = ResType<T, TGetAutoFarmingResponse, WsGetAutoFarmingMessage>;
  return agent.sendMessage<R>(chia_full_node_service, get_auto_farming_command);
}
```
Removed `data: TGetAutoFarmingRequest` because it's always an empty object (`{}`)